src/zjs_console.c

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2016, Intel Corporation.
+#ifdef BUILD_MODULE_CONSOLE
+
+#include "zjs_common.h"
+#include "zjs_util.h"
+
+#ifdef ZJS_LINUX_BUILD
+#define STDERR_PRINT(s, ...) fprintf(stderr, s, __VA_ARGS__)
+#define STDOUT_PRINT(s, ...) fprintf(stdout, s, __VA_ARGS__)
+#else
+#define STDERR_PRINT(s, ...) PRINT(s, __VA_ARGS__)
+#define STDOUT_PRINT(s, ...) PRINT(s, __VA_ARGS__)
+#endif
+
+static jerry_value_t console_log(const jerry_value_t function_obj,
+                                 const jerry_value_t this,
+                                 const jerry_value_t argv[],
+                                 const jerry_length_t argc)
+{
+    if (!jerry_value_is_string(argv[0])) {
+        ERR_PRINT("first parameter must be a string\n");
+        return ZJS_UNDEFINED;
+    }
+    jerry_size_t jlen = jerry_get_string_size(argv[0]);
+    char buffer[jlen + 1];
```

Comment:
We need to limit how much stack we use for this in order to avoid stack overflow and crash.